packages/pipeline-editor/src/PipelineSubmissionDialog.tsx

```diff
@@ -62,20 +58,16 @@ export class PipelineSubmissionDialog extends React.Component<IProps, IState> {
     const { schema, runtimes } = this.props;
 
     this.setState({
-      displayedRuntimeOptions: this.updateRuntimeOptions(
-        this.state.selectedRuntimePlatform
-      ),
       validSchemas: PipelineService.filterValidSchema(runtimes, schema)
     });
   }
 
   render(): React.ReactNode {
     const { name } = this.props;
-    const {
-      displayedRuntimeOptions,
-      selectedRuntimePlatform,
-      validSchemas
-    } = this.state;
+    const { selectedRuntimePlatform, validSchemas } = this.state;
+    const displayedRuntimeOptions = this.updateRuntimeOptions(
```

Comment:
maybe just rename this to `getRuntimeOptions` since this doesn't update anything anymore, but other than that I think this looks good to go